libyear-gradle-plugin/src/main/kotlin/com/libyear/traversal/DependencyTraversal.kt

```diff
@@ -7,7 +7,8 @@ import org.gradle.api.artifacts.result.ResolvedDependencyResult
 
 class DependencyTraversal private constructor(
   private val visitor: DependencyVisitor,
-  private val maxTransitiveDepth: Int?
+  private val maxTransitiveDepth: Int?,
+  private val excludedPackages: Set<String>
```

Comment:
I see why we would call that `excludedPackages` - because for most projects, the Java package names of the source code mirror the Maven artifact names in some way.

However, in Gradle terms, the concept of "package" does not exist. If I am not mistaken, "org.slf4j:slf4j" the part before the colon is the artifact's _group_, and after the colon follows the artifact's _name_. Together, the artifact's group and name form the name of a _module_.
Ref: `org.gradle.api.artifacts.ModuleIdentifier`

That's why I would suggest to rename to `excludedModules`.